htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -2136,6 +2136,25 @@ public function doCollectOneCollector($mode = 0)
 								} else {
 									$projectid = $objectemail->id;
 								}
+
+								if (get_class($objectemail) == 'Ticket') {
+									$changeonticket_references = false;
+									$trackid = $objectemail->track_id;
```

Comment:
Do we need this line ?

Reply:
You we do need this line because if the track id isn't found in the mail, then it recreate a ticket after the mail was collected even if a ticket was already created with an earlier email